src/uct/ib/mlx5/gga/gga_mlx5.c

```diff
@@ -509,6 +510,13 @@ static UCS_CLASS_INIT_FUNC(uct_gga_mlx5_iface_t,
                               &config->super.super, &config->rc_mlx5_common,
                               &init_attr);
 
+    status = uct_rc_mlx5_dp_ordering_ooo_init(
+            &self->super, UCT_IB_MLX5_MD_FLAG_DP_ORDERING_OOO_RW_RC,
+            &config->rc_mlx5_common, "gga");
+    if (status != UCS_OK) {
+        return UCS_ERR_INVALID_PARAM;
```

Comment:
why not `return status;`? 
the same to other usages - maybe `return UCS_ERR_INVALID_PARAM;` from `uct_rc_mlx5_dp_ordering_ooo_init`?

Reply:
fixed